src/Cache/CacheEngine.php

```diff
@@ -307,10 +306,7 @@ public function add(string $key, $value): bool
      * @param string $group name of the group to be cleared
      * @return bool
      */
-    public function clearGroup(string $group): bool
-    {
-        return false;
-    }
+    abstract public function clearGroup(string $group): bool;
```

Comment:
All engines have their own implementation for this method so this implementation was never used.